lib/core-server/package.json

```diff
@@ -39,9 +39,11 @@
   },
   "dependencies": {
     "@babel/core": "^7.12.10",
+    "@babel/plugin-transform-template-literals": "^7.12.1",
     "@babel/preset-react": "^7.12.10",
     "@storybook/addons": "6.2.0-alpha.32",
     "@storybook/builder-webpack4": "6.2.0-alpha.32",
+    "@storybook/core-client": "6.2.0-alpha.32",
```

Comment:
Why do we need this?

Reply:
As it's used as a default value in https://github.com/storybookjs/storybook/blob/70f9e6494dc4cfa25fb6cedda1a4b644d48d58f6/lib/core-server/src/presets/manager-preset.ts#L16